### PR TITLE
Add docker authz and selinux

### PR DESCRIPTION
**Pull request by wallies**

### CODE REVIEW

File: terraform/aws/public-cloud/main.tf [hunk 1]
```diff
@@ -93,14 +93,14 @@ module "kube_kubelet_certs" {
 }
 
 module "kube_admin_cert" {
-  source                = "github.com/Capgemini/tf_tls/kubernetes/admin"
+  source                = "github.com/Capgemini/tf_tls/kubernetes//admin"
```
Comment (tayzlor) [inline terraform/aws/public-cloud/main.tf]: the extra slashes here and around can go

**Comment by tayzlor** [comment]
needs a rebase

File: roles/docker/templates/policy.json.j2 [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{"name":"policy_1","users":["core", "root"],"actions":[""]}
+{"name":"policy_2","users":["service_account"],"actions":["container_logs","container_top"]}
+{"name":"policy_3","users":["kube"],"actions":["container"], "readonly":true }
```
Comment (enxebre) [inline roles/docker/templates/policy.json.j2]: Does this mean if a user is authenticate against kube wont be able to talk to docker? as he wouldn't be in any of these policies?

**Comment by enxebre** [comment]
As someone said once code lgtm if it works, I think we should probably do this toggable via env variable

File: roles/docker/templates/policy.json.j2 (hunk 2 above)
Comment (wallies) [inline roles/docker/templates/policy.json.j2]: kube is the service account we have at the moment for the default namespace and it can only read containers

**Pull request closed without merging**